eva-accession-core/src/test/resources/accession-test.properties

```diff
@@ -1,3 +1,5 @@
 accessioning.instanceId=test-instance-01
```

Comment:
Should we rename this file as `application.properties` until we have the need for multiple test properties?

Reply:
I did this to avoid confusions with the test `.properties` of other submodules